versions/3.0.3.md

```diff
@@ -317,15 +317,15 @@ url: https://www.apache.org/licenses/LICENSE-2.0.html
 
 #### <a name="serverObject"></a>Server Object
 
-An object representing a Server.
+An object representing a single server or a group of servers sharing a common URL structure.
```

Comment:
In this case Server Object represents a logical origin server, not whether there is one more physical machines.  Saying that server could mean one or more is not really adding any clarity in my opinion. Once server variables are resolved it only points to one logical server.